cloudcix_primitives/bridgemain.py

```diff
@@ -0,0 +1,436 @@
+        3031: 'Failed to connect to the local host for start_service payload: ',
+        3032: 'Failed to run start_service payload on the local host. Payload exited with status ',
+
+    }
+
+    # template data for required script files
+    template_data = {
+        'address_range': address_range,
+        'bridge': bridge,
+        'down_script_path': down_script_path,
+        'up_script_path': up_script_path,
+    }
+
+    # Templates
+    # down script
+    template = JINJA_ENV.get_template('bridgemain/down.sh.j2')
+    template_verified, template_error = check_template_data(template_data, template)
+    if not template_verified:
+        return False, f'3002: {messages[3002]}'
+    down_script = template.render(**template_data)
+    # up script
+    template = JINJA_ENV.get_template('bridgemain/up.sh.j2')
+    template_verified, template_error = check_template_data(template_data, template)
+    if not template_verified:
+        return False, f'3003: {messages[3003]}'
+    up_script = template.render(**template_data)
+    # service file
+    template = JINJA_ENV.get_template('bridgemain/interface.service.j2')
+    template_verified, template_error = check_template_data(template_data, template)
+    if not template_verified:
+        return False, f'3004: {messages[3004]}'
+    service_file = template.render(**template_data)
+
+    def run_host(host, prefix, successful_payloads):
+        fmt = HostErrorFormatter(
+            host,
+            {'payload_message': 'STDOUT', 'payload_error': 'STDERR'},
+            successful_payloads,
+        )
+
+        payloads = {
+            'create_down_script': "\n".join([
+                f'tee {down_script_path} <<EOF && chmod 744 {down_script_path}',
+                down_script,
+                "EOF"
+            ]),
+            'create_up_script': "\n".join([
+                f'tee {up_script_path} <<EOF && chmod 744 {up_script_path}',
+                up_script,
+                "EOF"
+            ]),
+            'create_service_file': "\n".join([
+                f'tee {service_file_path} <<EOF && chmod 744 {service_file_path}',
+                service_file,
+                "EOF"
+            ]),
+            'find_service': f'systemctl status bridgemain_{bridge}.service',
+            'start_service': f'systemctl restart bridgemain_{bridge}.service && '
+                             f'systemctl enable bridgemain_{bridge}.service',
+            'reload_services': 'systemctl daemon-reload',
+        }
+
+        ret = comms_lsh(payloads['find_service'])
+        if ret["channel_code"] != CHANNEL_SUCCESS:
+            return False, fmt.channel_error(ret, f'{prefix + 1}: {messages[prefix + 1]}'), fmt.successful_payloads
+        create_service = True
+        if ret["payload_code"] != SUCCESS_CODE:
+            fmt.payload_error(ret, f'{prefix + 2}: {messages[prefix + 2]}'), fmt.successful_payloads
```

Comment:
For build, why is there an error recorded here? It would be expected that the service is not found during build.